app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -57,9 +58,13 @@ public DataHandler(Context context) {
         else {
             contactProvider = null;
         }
-
-        if (prefs.getBoolean("enable-search", true)) {
-            providers.add(new SearchProvider(context));
+        switch(prefs.getString("searchEngine", "none")) {
```

Comment:
Google is the default: https://github.com/losingkeys/KISS/blob/a55781007c234ee886fd2919ec94684453a03a7d/app/src/main/res/xml/preferences.xml#L49